lib/storage/client-local.js

```diff
@@ -22,5 +22,9 @@ export function get(key) {
 export function set(key, value) {
     try {
         storage[`${prefixKey}${key}`] = JSON.stringify(value);
+
+        const event = new Event('tjek_shopping_list_update');
+
+        window.dispatchEvent(event);
```

Comment:
It's not okay for everything in the SDK that calls `clientStore.set()` to dispatch this event